<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Copilot</class>
 <widget class="QWidget" name="Copilot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1921</width>
        <height>1081</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>1621</width>
        <height>751</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="5" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QFrame" name="frame_7">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="2">
           <widget class="QLabel" name="SecondaryCameraView1">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Policy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="SecondaryCameraView2">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QWidget" name="pressure" native="true">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="Background">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="RPB_BACK">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 255);
border:none;
border-radius: 59px;</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::NoFrame</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="RPB_PATH">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.823928 rgba(85, 255, 255, 255), stop:0.828442 rgba(0, 0, 124, 255));
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Shape::NoFrame</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_19">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="RPB_Front">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>100</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>100</width>
                       <height>100</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>4</pointsize>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(0, 0, 127);
border: none;
border-radius:50;</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::Shape::NoFrame</enum>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="rpb_kpa">
                        <property name="maximumSize">
                         <size>
                          <width>100</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>12</pointsize>
                         </font>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                        </property>
                        <property name="text">
                         <string>0 kPa</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="rpb_perc">
                        <property name="maximumSize">
                         <size>
                          <width>100</width>
                          <height>40</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>12</pointsize>
                         </font>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
                        </property>
                        <property name="text">
                         <string>0%</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QWidget" name="ambient_temp" native="true">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>250</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>250</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="Background2">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <widget class="QFrame" name="Thermo2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>105</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 0, 255);
border: 2px solid;
border-radius: 20px;
border-color: rgb(255, 255, 255);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <widget class="QLabel" name="temp_value">
              <property name="geometry">
               <rect>
                <x>5</x>
                <y>5</y>
                <width>30</width>
                <height>28</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;
background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="text">
               <string>19Â°</string>
              </property>
             </widget>
            </widget>
            <widget class="QFrame" name="Thermo">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>10</y>
               <width>20</width>
               <height>101</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">border:1px solid;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(14, 14, 255, 255), stop:1 rgba(255, 0, 0, 255));
border-top-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-bottom-color: rgb(0, 0, 255);
border-radius: 10px;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <widget class="QProgressBar" name="temp_bar">
              <property name="geometry">
               <rect>
                <x>6</x>
                <y>-20</y>
                <width>8</width>
                <height>135</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">border: none;
border-radius: 5px;
background-color: rgba(255, 255, 255, 0);</string>
              </property>
              <property name="value">
               <number>77</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="format">
               <string/>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="MainCameraView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
       </rect>
      </property>
      <property name="text">
       <string>lol</string>
      </property>
     </widget>
     <zorder>MainCameraView</zorder>
     <zorder>frame</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
